### PR TITLE
Remove TABLE_HEADING_QUANTITY override

### PR DESCRIPTION
**Pull request by scottcwilson**
This value is defined in english.php so the override won't work

### CODE REVIEW

**Comment by scottcwilson** [comment]
Fixes #2379

File: includes/languages/english/account.php [hunk 1]
```diff
@@ -17,7 +17,6 @@
 define('TABLE_HEADING_ORDER_NUMBER', 'No.');
 define('TABLE_HEADING_SHIPPED_TO', 'Ship To');
 define('TABLE_HEADING_STATUS', 'Status');
-define('TABLE_HEADING_TOTAL', 'Total');
```
Comment (zcwilt) [inline includes/languages/english/account.php]: Was this intentional

Comment (scottcwilson) [reply includes/languages/english/account.php]: Yes, same issue with that define.  But I can break it out if you prefer.

**Comment by zcwilt** [comment]
looking at your issue #2379 I thought the intention was to remove the defines from the main language file.

**Comment by scottcwilson** [comment]
I thought about doing it that way but realized the other way around was a better path.

**Comment by zcwilt** [comment]
However it does mean that there is now no way to override those defines on other pages.

**Comment by scottcwilson** [comment]
In the spirit of  issue #2008 I would prefer to move this define to the top and reduce duplication.  If a different string is truly required, people can create a new define. If we were to move this define out of the top level, it would necessitate creating it at a number of new places which require the top level define.

**Comment by scottcwilson** [comment]
I think this approach - consolidating string definitions and assuming they only have one value throughout the site - will make upgrades easier and faster for people.

**Pull request merged**